package.json

```diff
@@ -23,9 +22,9 @@
     "filepond-plugin-file-validate-size": "^1.0.4",
     "filepond-polyfill": "^1.0.2",
     "object-assign": "4.1.1",
-    "react": "^16.8.1",
+    "react": "^18.2.0",
```

Comment:
React in all it's up-to-date glory. I am not too familiar with all its new stuff, but our stuff will just work, so it's worth doing this.